src/webui/www/private/scripts/client.js

```diff
@@ -622,7 +622,7 @@ window.addEvent('load', function() {
         $('DHTNodes').set('html', 'QBT_TR(DHT: %1 nodes)QBT_TR[CONTEXT=StatusBar]'.replace("%1", serverState.dht_nodes));
 
         // Statistics dialog
-        if (document.getElementById("statisticspage")) {
+        if (document.getElementById("statisticspage") && document.getElementById("statisticsContent")) {
```

Comment:
Why checking for `statisticspage` is not sufficient?
Does it needs 2 checks? I mean what if you wrote: `if (document.getElementById("statisticsContent")`?

Reply:
Yes, it should be sufficient like that, but it made sense to me to keep the other part. It is supposed to signal the intent that, like the other windows, we wait for the page itself, but unlike the others, we must also wait for the content to arrive before manipulating it.

I'd also say this way it protects us from some weird situation where the window itself does not get created for some reason, but honestly that is not a realistic concern.

Anyway, if you think it is better with the simpler condition, I'll change it.

Reply:
>Anyway, if you think it is better with the simpler condition, I'll change it.

I think it is. After seeing the `id` is required be unique on a page, it makes no point to check for `statisticspage` if it isn't sufficient.

Reply:
@Chocobo1 fixed.